tests/src/test/java/alluxio/client/fs/ConcurrentFileSystemMasterCreateTest.java

```diff
@@ -275,10 +275,14 @@ private void runLoadMetadata(WriteType writeType, boolean useSinglePath, boolean
 
     int uniquePaths = useSinglePath ? 1 : numThreads;
 
+    if (writeType != WriteType.CACHE_THROUGH) {
```

Comment:
where does the `4` come from? Is it intended that this get overridden by the `if (listParentDir)` block?

Reply:
I think the 4 is just a buffer for loaded machines? And yes, we want the block below to overwrite this.

Reply:
Sounds good. Done.